File: JavaScript/8js 2/include.js
/*
|--------------------------------------------------------------------------
| Enumerable Magic - Does My List Include This?
|--------------------------------------------------------------------------
|
*/

function include(arr, item) {
   return arr.includes(item);
}


function include(arr, item) {
   return arr.indexOf(item) > -1;
}


function include(arr, item) {
   return arr.some((n) => n === item);
}


console.log(include([1, 2, 3, 4], 3), true);
console.log(include([1, 2, 4, 5], 3), false);
console.log(include([], 3), false);

File: Python/8py 2/include.py
"""
|--------------------------------------------------------------------------
| Enumerable Magic - Does My List Include This?
|--------------------------------------------------------------------------
|
"""

def include(arr, item):
   return item in arr


include = list.__contains__


def include(arr,item):
   for i in arr:
      if i == item:
         return True
   return False


print(include([1,2,3,4], 3), True)
print(include([1,2,4,5], 3), False)